pulsar-common/src/main/java/org/apache/pulsar/common/protocol/Commands.java

```diff
@@ -1525,11 +1525,6 @@ private static ByteBufPair serializeCommandSendWithSize(BaseCommand cmd, Checksu
         return command;
     }
 
-    public static ByteBuf addBrokerEntryMetadata(ByteBuf headerAndPayload,
```

Comment:
What is the reason for removing this one?

Reply:
There is no other code that invokes this method.

Reply:
This could be used by 3rd party clients. It is part of our public API.

Reply:
Unfortunately we have protocol handlers and we can't easily drop 'public' classes and methods

Reply:
@eolivelli Do you know which protocol handler uses this method? I'm curious how the protocol handler works with the method, add broker entry metadata is only used by the managed ledger internal. If the protocol handler wants to introduce broker entry metadata changes, it should implement a new `BrokerEntryMetadataInterceptor`.

IMO, essentially, this is not a public API at the beginning of the design, It's just that all the pulsar command-related methods are put here.